react-native/android/analytics.gradle

```diff
@@ -57,8 +57,9 @@ class SendAnalyticsTask extends DefaultTask {
     def sendAnalytics() {
        try {
             def env = System.getenv()
-            def disableAnalytics= env['REALM_DISABLE_ANALYTICS']
-            if (disableAnalytics == null || disableAnalytics != "true") {
+            def disableAnalytics = env['REALM_DISABLE_ANALYTICS'] != null
+            def isCI = env['CI'] != null
+            if (!disableAnalytics && !isCI) {
```

Comment:
This would have the unfortunate side-effect of disabling analytics when `REALM_DISABLE_ANALYTICS=false` which was not the semantics that the previous implementation had, I don't know how we document this, but perhaps this is not what you ment?

Reply:
It matches the semantics for iOS and Node/Electron so it made sense to align on all platforms:

https://github.com/realm/realm-js/blob/48144ae85fe02d27663101a1fef9c927e6cca29a/lib/submit-analytics.js#L99

https://github.com/realm/realm-js/blob/48144ae85fe02d27663101a1fef9c927e6cca29a/react-native/ios/RealmReact/RealmAnalytics.mm#L230

I feel like just setting `REALM_DISABLE_ANALYTICS` to any value should stop analytics submission. For example, realm-js's own CI workflow sets `REALM_DISABLE_ANALYTICS` to `1` rather than `true`, and I assume many users might feel these are interchangeable. Rather than special truthy and falsy values, it's more straightforward to just check for existence. If a user wants to enable analytics, they should just not set the environment variable.

Reply:
For the integration tests, we have https://github.com/realm/realm-js/blob/master/.github/workflows/integration-tests.yml#L4 (`REALM_DISABLE_ANALYTICS` is not `null`).

Moreover, we should set `REALM_DISABLE_ANATYTICS` for all test runner in `Jenkinsfile` (as we do for MacOS: https://github.com/realm/realm-js/blob/master/Jenkinsfile#L486).